scoop_protocol/scoop_protocol.go

```diff
@@ -43,6 +43,11 @@ type LoadCheckResponse struct {
 	ManifestURL string
 }
 
+//Scoop healthcheck for ingester healthcheck json unmarshal
+type ConnError struct {
+	RedshiftDBConnError, IngesterDBConnError *string
```

Comment:
In the rest of this file, we put each field on its own line. I would suggest doing the same here.